lib/php-openid/Auth/OpenID/Consumer.php

```diff
@@ -1263,7 +1263,7 @@ function _verifyDiscoveryServices($claimed_id,
             foreach ($to_match_endpoints as $to_match_endpoint) {
                 $result = $this->_verifyDiscoverySingle($endpoint, $to_match_endpoint);
 
-                if ($result && !Auth_OpenID::isFailure($result)) {
+                if (!Auth_OpenID::isFailure($result)) {
```

Comment:
can you please document this modification within lib/php-openid/README.txt too. Thanks

Reply:
Done.